person.rb

```diff
@@ -0,0 +1,22 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission = 'true')
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def of_age?
+    @age >= 18
+  end
+
+  def can_use_services?
+    return true if of_age? || @parent_permission
+
+    false
```

Comment:
- [ ] [OPTIONAL] including the false in this case is redundant, the method will return false without this line as well.